CONTRIBUTORS.yaml

```diff
@@ -507,6 +507,10 @@ Camila-goclowski:
     email: camila.goclowski@gmail.com
     linkedin: camila-goclowski
 
+carloscheemendonca:
+    name: Carlos Chee Mendonça
+    joined: 2025-01
```

Comment:
@carloscheemendonca please feel free to edit or add to this entry as you see fit